source/views/stoprint/print-release.js

```diff
@@ -0,0 +1,244 @@
+		} else {
+			const heldJobMatch = response.value.find(heldJob =>
+				heldJob.id.startsWith(jobId),
+			)
+			if (heldJobMatch) {
+				this.setState(() => ({heldJob: heldJobMatch}))
+			} else {
+				showGeneralError(this.returnToJobsView)
+			}
+		}
+	}
+
+	requestCancel = () => {
+		const {job} = this.props.navigation.state.params
+		const prompt = `Are you sure you want to cancel printing ${
+			job.documentName
+		}? This cannot be undone.`
+		Alert.alert('Print Job Cancellation Confirmation', prompt, [
+			{text: 'Cancel', style: 'cancel'},
+			{text: 'OK', onPress: this.cancelJob},
+		])
+	}
+
+	requestRelease = () => {
+		const {job, printer} = this.props.navigation.state.params
+		const prompt = `Are you sure you want to print ${job.documentName} to ${
+			printer.printerName
+		}?`
+		Alert.alert('Print Job Release Confirmation', prompt, [
+			{text: 'Cancel', style: 'cancel'},
+			{text: 'OK', onPress: this.releaseJob},
+		])
+	}
+
+	releaseJob = async () => {
+		this.setState(() => ({status: 'printing'}))
+		const {printer, username, job} = this.props.navigation.state.params
+		const {heldJob} = this.state
+		if (!heldJob) {
+			showGeneralError(this.returnToJobsView)
+			return
+		}
+		const response: ReleaseResponseOrErrorType = await releasePrintJobToPrinterForUser(
+			{
+				jobId: heldJob.id,
+				printerName: printer.printerName,
+				username: username,
+			},
+		)
+		if (response.error) {
+			Alert.alert(
+				'Error Releasing Job',
+				'We encountered a problem while trying to release your job to the printer. Please try again or release your job at the printer itself.',
+				[{text: 'OK', onPress: this.returnToJobsView}],
+			)
+		} else {
+			Alert.alert(
+				'Job Successfully Released',
+				`Document ${job.documentName} is printing at ${printer.printerName}.`,
+				[{text: 'OK', onPress: this.returnToJobsView}],
+			)
+		}
+	}
+
+	cancelJob = async () => {
+		this.setState(() => ({status: 'cancelling'}))
+		const {username, job} = this.props.navigation.state.params
+		const {heldJob} = this.state
+		if (!heldJob) {
+			showGeneralError(this.returnToJobsView)
+			return
+		}
+		const response: CancelResponseOrErrorType = await cancelPrintJobForUser(
+			heldJob.id,
+			username,
+		)
+		if (response.error) {
+			Alert.alert(
+				'Error Cancelling Job',
+				'We encountered a problem while trying to cancel your job. Please try again or cancel your job at the printer itself.',
+				[{text: 'OK', onPress: this.returnToJobsView}],
+			)
+		} else {
+			Alert.alert(
+				'Job Successfully Cancelled',
+				`Document ${job.documentName} has been removed from your print queue.`,
+				[{text: 'OK', onPress: this.returnToJobsView}],
+			)
+		}
+	}
+
+	returnToJobsView = () => {
+		this.props.navigation.navigate('PrintJobsView')
+	}
+
+	render() {
+		const {job, printer} = this.props.navigation.state.params
+		const {status} = this.state
+		const actionAvailable = status !== 'complete' && printer
+		return (
+			<Container>
```

Comment:
Can you change this to just a normal ScrollView and put the style on `contentContainerStyle` instead of nesting the ScrollView inside of a View, please?